src/mpi/mpi_caf.c

```diff
@@ -4454,39 +4454,36 @@ PREFIX (event_wait) (caf_token_t token, size_t index,
 		     int until_count, int *stat,
 		     char *errmsg, int errmsg_len)
 {
-  int ierr=0,count=0,i,image=caf_this_image-1;
-  int *var=NULL,flag,old=0;
-  int newval=0;
+  int ierr = 0, count = 0, i, image = caf_this_image - 1;
+  int *var = NULL, flag, old = 0;
+  int newval = 0;
```

Comment:
Sorry, couldn't read myself, therefore went for adapting to the style. Should have done so in two commits, to separate stylistic from programmatic changes.